lib/connect_proto/src/order.proto

```diff
@@ -72,11 +74,12 @@ message Order {
     enum ResponseFlag {
       /* Increasing specificity, values incorporate previous options */
       /* HL7 Table 0121 */
-      ACKNOWLEDGEMENT = 0;
+      UNKNOWN = 0;
       EXCEPTIONS = 1;
       REPLACEMENTS = 2;
       ASSOCIATED_SEGMENTS = 3;
       CONFIRMATIONS = 4;
+      ACKNOWLEDGEMENT = 5;
```

Comment:
Not sure we can push ACKNOWLEDGEMENT to 5 given the comment. Might have to shift them all